com.github.tenderowl.norka.json

```diff
@@ -49,28 +50,28 @@
                 {
                     "type": "git",
                     "url": "https://gitlab.gnome.org/GNOME/libhandy.git",
-                    "tag" : "1.5.0"
+                    "tag": "1.8.2"
```

Comment:
I'd be better if all the tags had commit hashes also

Reply:
Awesome, didn't look into runtime for a while.